src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs

```diff
@@ -69,9 +69,38 @@ public Resume GetById(Guid id)
             return resume;
         }
 
-        public void Update(Guid id, Resume updateObject)
+        public Resume Update(Resume updateObject)
         {
-            throw new NotImplementedException();
+            var sql = "INSERT INTO ResumeSkill ( IdResume, IdSkill ) VALUES ( @idResume, @idSkill )";
+            foreach (var skill in updateObject.Skills)
+            {
+                using (IDbConnection db = new SqlConnection(this.connectionString))
+                {
+                    db.Open();
+                    DynamicParameters parameter = new DynamicParameters();
+                    parameter.Add("@idResume", updateObject.Id, DbType.Int32);
+                    parameter.Add("@idSkill", skill.Id, DbType.Int32);
+                    db.Execute(sql, parameter);
+                }
+            }
+
+            return updateObject;
+        }
+
+        public IEnumerable<Skill> AddSkills(IEnumerable<Skill> skills)
```

Comment:
please double check if it's possible to use Skill[] for the input param, no need an ienumerable from my point of view